.gitignore

```diff
@@ -351,5 +351,8 @@ MigrationBackup/
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
 
-# Ignore default autorest generated folder in teh root only.
-/generated
\ No newline at end of file
+# Ignores generated files from samples
+samples/**/*.yaml
+!samples/Xkcd/xkcd.yaml
+samples/**/*.cs
```

Comment:
What settings do you use to have it generate under the samples folder? Or does that just happen when you use the readme.md as the configuration?

Reply:
Oh I see the output-folder being defined in the readmes now. Cool that works.